spyder/plugins/editor/extensions/closebrackets.py

```diff
@@ -35,10 +29,10 @@ def _on_key_pressed(self, event):
         if event.isAccepted():
             return
 
-        key = event.key()
-        if key in self.BRACKETS_CHAR and self.enabled:
+        char = event.text()
```

Comment:
I would just add a comment here to highlight why we use `event.text()`

Something like. `# See spyder-ide/spyder#9749` to keep the code informed for future devs